src/components/Contacts/ContactDetails/ContactDetails.tsx

```diff
@@ -9,27 +10,76 @@ interface Props {
   onClose: () => void;
 }
 
+interface ContactTabsProps {
+  value: number;
+  onChange: (event: React.ChangeEvent, newIndex: number) => void;
+}
+
+const ContactDetailsWrapper = styled(Box)(({}) => ({
+  width: '100%',
+}));
+
+const ContactTabsWrapper = styled(Box)(({}) => ({
+  width: '100%',
+  backgroundColor: 'transparent',
+  boxShadow: 'none',
+  borderBottom: '1px solid #DCDCDC',
+}));
+
+const ContactTabs = withStyles({
```

Comment:
I somehow needed to access "indicator" on the tabs and style it.  This is what the sample in the MaterialUI docs used... I'm not sure how to get a similar effect with `styled`.

Reply:
This seems to work:
```diff
diff --git a/src/components/Contacts/ContactDetails/ContactDetails.tsx b/src/components/Contacts/ContactDetails/ContactDetails.tsx
index d3eb2ad..4ae673a 100644
--- a/src/components/Contacts/ContactDetails/ContactDetails.tsx
+++ b/src/components/Contacts/ContactDetails/ContactDetails.tsx
@@ -1,4 +1,4 @@
-import { Box, styled, Tab, Tabs, withStyles } from '@material-ui/core';
+import { Box, styled, Tab, Tabs } from '@material-ui/core';
 import React from 'react';
 import { useTranslation } from 'react-i18next';
 import theme from '../../../theme';
@@ -10,11 +10,6 @@ interface Props {
   onClose: () => void;
 }
 
-interface ContactTabsProps {
-  value: number;
-  onChange: (event: React.ChangeEvent, newIndex: number) => void;
-}
-
 const ContactDetailsWrapper = styled(Box)(({}) => ({
   width: '100%',
 }));
@@ -26,11 +21,9 @@ const ContactTabsWrapper = styled(Box)(({}) => ({
   borderBottom: '1px solid #DCDCDC',
 }));
 
-const ContactTabs = withStyles({
-  root: {
-    width: '100%',
-    minHeight: 40,
-  },
+const ContactTabs = styled(Tabs)(({}) => ({
+  width: '100%',
+  minHeight: 40,
   indicator: {
     display: 'flex',
     '& > span': {
@@ -39,9 +32,7 @@ const ContactTabs = withStyles({
       backgroundColor: '#FFCF07',
     },
   },
-})((props: ContactTabsProps) => (
-  <Tabs {...props} TabIndicatorProps={{ children: <span /> }} />
-));
+}));
 
 const ContactTab = styled(Tab)(({}) => ({
   textTransform: 'none',
@@ -72,7 +63,11 @@ export const ContactDetails: React.FC<Props> = ({
         contactId={contactId}
       />
       <ContactTabsWrapper>
-        <ContactTabs value={selectedTabIndex} onChange={handleChange}>
+        <ContactTabs
+          value={selectedTabIndex}
+          onChange={handleChange}
+          TabIndicatorProps={{ children: <span /> }}
+        >
           <ContactTab label={t('Tasks')} />
           <ContactTab label={t('Donations')} />
           <ContactTab label={t('Referrals')} />
```

I think you can make these changes locally by pasting them into file (ex `patch.diff`)  and applying them with `git apply patch.diff`.